Makefile

```diff
@@ -159,11 +159,6 @@ retail-getting-started-pdf: suma xml-retail
 #### Retail Migration
 xml-retail-migration: xml/book_retail-migration.xml retail-images
 
-retail-images: adoc/images/retail/*
-	mkdir -p images/src
-	(mkdir -p images/src/png; cd images/src/png; ln -sf ../../../adoc/images/retail/*.png .)
-	(mkdir -p images/src/svg; cd images/src/svg; ln -sf ../../../adoc/images/retail/*.svg .)
-
```

Comment:
The point is that this rule is twice in the makefile... There is no need to duplicate it for Retail and Retail migration since those are strictly identical.

Reply:
Yeah, I get it. What I'm saying is that those two sections ("Retail" and "Retail Migration") should be merged into one section called "Retail" which would include both books.

Reply:
Much better! Thanks :)